src/service/video-manager-impl.js

```diff
@@ -334,6 +419,25 @@ class VideoEntry {
     }
   }
 
+  /* Docking Behaviour */
+
+  /**
+   * Called when a dockable video is built.
+   * @private
+   */
+  dockableVideoBuilt_() {
+    this.video.element.classList.add('i-amphtml-dockable-video');
+
+    const vidRect = this.video.element./*OK*/getBoundingClientRect();
```

Comment:
**Note:** We decided to go with `getBoundingClientRect()` since `this.video.element.getLayoutBox()` wasn't always giving the correct dimensions on build time.